oximeter/db/src/client/mod.rs

```diff
@@ -1147,6 +1186,24 @@ async fn handle_db_response(
     }
 }
 
+/// A trait allowing a [`Client`] to use the sql methods directly.
+///
+/// In general, clients should use oxql, but in many tests we use the SQL methods.
+/// For integration test purposes we need to make them public. However, we don't
+/// want to expose these functions for general consumption without extra clarity.
```

Comment:
The short answer is I don't think this is possible if we want to read the `_local` tables directly. `oxql` only reads via the distributed path.

Reply:
Ahhhhhhh, hm. Perhaps we don't _need_ to read from `_local` tables directly? Anyway, this doesn't really affect this PR, but I guess just something to keep in mind

Reply:
Yeah, we don't actually use this right now at all. Maybe I should just get rid of it. I added it before I decided to use oxql directly. This would obviate making the change above to also check those `_local` tables, but I think that's ok.

Reply:
I removed this capability in [417975e](https://github.com/oxidecomputer/omicron/pull/6131/commits/417975e01f7d12e8bdfbf9efb472350addb38662)